packages/ai/src/ai/modules/task/commands/cmd_account.py

```
@@ -0,0 +1,1243 @@
+            ValueError: If ``teamId`` or ``name`` is missing.
+        """
+        # A Zitadel token is required so that interactive login is needed to
+        # create an API key — preventing key theft via a compromised API key.
+        self.require_zitadel_auth()
+        user_id = self._account_info.userId
+        team_id = args.get('teamId')
+        name = args.get('name')
+
+        # Both teamId and name are mandatory — reject early with a clear message.
+        if not team_id or not name:
+            raise ValueError('teamId and name are required')
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)